src/wasm-interpreter.h

```diff
@@ -49,13 +49,18 @@ extern Name WASM, RETURN_FLOW;
 // in control flow.
 class Flow {
 public:
-  Flow() = default;
-  Flow(Literal value) : value(value) {}
-  Flow(Name breakTo) : breakTo(breakTo) {}
+  Flow() { values.push_back(Literal()); }
+  Flow(Literal value) { values.push_back(value); }
+  Flow(Name breakTo) : breakTo(breakTo) { values.push_back(Literal()); }
 
-  Literal value;
+  SmallVector<Literal, 1> values;
   Name breakTo; // if non-null, a break is going on
 
+  Literal& getValue() {
+    assert(values.size() == 1);
```

Comment:
We need multiple values in the flow to handle the case where a `tuple.make` or a multivalue function call is interpreted. For most cases, the interpreter can assume due to validation that there is only a single value in the flow, so those cases can use this helper function instead of having to examine `values` directly. I'm open to other opinions on how this could be made cleaner.